### PR TITLE
[#353] update to next dev version

### PR DESCRIPTION
**Pull request by ewilkins-csi**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -12,19 +12,19 @@ on:
   workflow_dispatch:
     inputs:
       releaseVersion:
-        description: "The version you want to release aiSSEMBLE on."
+        description: "Release Version"
```
Comment (ewilkins-csi) [inline .github/workflows/release.yml:15]: I: These weren't super readable in the UI so updated to shorter names

File: DRAFT_RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,44 +1,21 @@
 # Major Additions
```
Comment (ewilkins-csi) [inline DRAFT_RELEASE_NOTES.md:1]: I: Draft release notes reset

File: foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/util/YamlUtils.java [hunk 3]
```diff
@@ -254,7 +263,7 @@ public static int getIndentSpaces(List<String> yamlFileContent, int startIndex)
         int index = startIndex;
         while (indentSpaces == 0) {
             String line = yamlFileContent.get(index).stripTrailing();
-            if (!StringUtils.isBlank(line)) {
+            if (!StringUtils.isBlank(line) && !isComment(line.trim()) && !isHelmFunction(line.trim())) {
```
Comment (ewilkins-csi) [inline foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/util/YamlUtils.java:266]: I: Comments and helm functions may not follow the standard YAML indentation

File: foundation/foundation-upgrade/src/main/resources/migrations.json [hunk 4]
```diff
@@ -1,66 +1,8 @@
 [
   {
-    "group": "v1-9-0-migrations",
+    "group": "v1-10-0-migrations",
```
Comment (ewilkins-csi) [inline foundation/foundation-upgrade/src/main/resources/migrations.json:3]: I: Reset migrations

File: extensions/extensions-helm/aissemble-kafka-chart/values.yaml [hunk 5]
```diff
@@ -2,7 +2,6 @@
 ## CONFIG | Kafka Configs
 ########################################
 kafka:
-  kafkaVersion: 3.8.0
```
Comment (csun-cpointe) [inline extensions/extensions-helm/aissemble-kafka-chart/values.yaml:5]: Q: no more version required?

**Review by csun-cpointe — APPROVED** [review]

File: extensions/extensions-helm/aissemble-kafka-chart/values.yaml (hunk 5 above)
Comment (ewilkins-csi) [reply extensions/extensions-helm/aissemble-kafka-chart/values.yaml:5]: This parameter is not valid in the base chart at all and we have no templates. So it's a completely unused property.

**Pull request merged**